client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -212,7 +234,24 @@ export default React.createClass( {
 		analytics.tracks.recordEvent( 'calypso_plugins_actions_update_plugin', {
 			site: this.props.sites[ 0 ].ID,
 			plugin: this.props.sites[ 0 ].plugin.slug,
-			selectedSite: this.props.sites[ 0 ].ID
+			selected_site: this.props.sites[ 0 ].ID
+		} );
+	},
+
+	handlePluginUpdatesMultiSite( event ) {
+		event.preventDefault();
+		this.props.sites.forEach( ( site ) => {
+			if ( site.canUpdateFiles && site.plugin && site.plugin.update && 'error' !== site.plugin.update ) {
```

Comment:
I think we need to check that there is something to update here. I think doing something like `site.plugin.update.new_version` because currently when we have autoupdate happening automatically we might be updating some plugins and then clicking the update link and updating the plugin again.